### ISSUE TITLE
Tune super-voxel size and SVB block size

### ISSUE THREAD

**Issue by XiaoWang-Github** [HPImaging/sv-mbirct#15]
We need to tune super-voxel size and SVB block size.

**Comment by sjkisner** [comment]
Yes, need to first make these as input parameters (currently they're fixed at compile time).  Then I'll work on a tool, probably on the python side, that runs a sweep for a specified image/sinogram size and outputs performance numbers.
